OpenSim/Simulation/SimbodyEngine/SimbodyEngine.h

```diff
@@ -185,6 +162,53 @@ OpenSim_DECLARE_CONCRETE_OBJECT(SimbodyEngine, Object);
     void convertQuaternionsToDirectionCosines(double aQ1, double aQ2, double aQ3, double aQ4, double rDirCos[3][3]) const;
     void convertQuaternionsToDirectionCosines(double aQ1, double aQ2, double aQ3, double aQ4, double *rDirCos) const;
 
+    //-------------------------------------------------------------------------
+    // DEPRECATED METHODS
+    //-------------------------------------------------------------------------
+    /** @name Deprecated */
+    // @{
+    /** <b>(Deprecated)</b> Use Frame::findLocationInGround() instead. */
+    DEPRECATED_14("use Frame::findLocationInGround() instead")
+    void getPosition(const SimTK::State& s, const PhysicalFrame &aBody, const SimTK::Vec3& aPoint, SimTK::Vec3& rPos) const;
+    DEPRECATED_14("use Frame::findVelocityInGround() instead")
+    void getVelocity(const SimTK::State& s, const PhysicalFrame &aBody, const SimTK::Vec3& aPoint, SimTK::Vec3& rVel) const;
+    DEPRECATED_14("use Frame::findAccelerationInGround() instead")
+    void getAcceleration(const SimTK::State& s, const PhysicalFrame &aBody, const SimTK::Vec3& aPoint, SimTK::Vec3& rAcc) const;
+    DEPRECATED_14("use Frame::getTransformInGround().R() instead")
+    void getDirectionCosines(const SimTK::State& s, const PhysicalFrame &aBody, double rDirCos[3][3]) const;
+    DEPRECATED_14("use Frame::getTransformInGround().R() instead")
+    void getDirectionCosines(const SimTK::State& s, const PhysicalFrame &aBody, double *rDirCos) const;
+    DEPRECATED_14("use Frame::getVelocityInGround()(0) instead")
+    void getAngularVelocity(const SimTK::State& s, const PhysicalFrame &aBody, SimTK::Vec3& rAngVel) const;
+    DEPRECATED_14("use Frame::getVelocityInGround()(0) instead")
```

Comment:
I wasn't quite sure what to do here because if you look at the implementation of the two methods ([here](https://github.com/opensim-org/opensim-core/blob/master/OpenSim/Simulation/SimbodyEngine/SimbodyEngine.cpp#L270) and [here](https://github.com/opensim-org/opensim-core/blob/master/OpenSim/Simulation/SimbodyEngine/SimbodyEngine.cpp#L283)) they seem to be implemented in exactly the same way. Simbody's doxygen [comment](https://simtk.org/api_docs/simbody/3.5/classSimTK_1_1MobilizedBody.html#ad07e64cc62fa2f76bd37894007abd378) suggests that they will both return the angular velocity in Ground frame. Thus, the true replacement is unfortunately the same function for both.